### PR TITLE
IMP account_fiscal_agent: fiscal position view; invoice confirmation

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: account_fiscal_agent/models/account_fiscal_position.py [hunk 1]
```diff
@@ -34,6 +34,10 @@ def map_agent_journal(self, journal):
         return journal
 
     with_fiscal_agent = fields.Boolean('With Fiscal Agent?')
+    fiscal_agent_company_id = fields.Many2one(
+        'res.company', "Fiscal agent company",
+        help="Company of the fiscal agent: new invoices will be created for "
+             "this company")
     fiscal_agent_account_ids = fields.One2many('account.fiscal.agent.account',
```
Comment (JayVora-SerpentCS) [inline account_fiscal_agent/models/account_fiscal_position.py:48]: This is a good add. We discussed internally for the same.

File: account_fiscal_agent/models/account_invoice.py [hunk 2]
```diff
@@ -8,7 +8,8 @@
 class account_invoice(models.Model):
     _inherit = 'account.invoice'
 
-    agent_invoice_id = fields.Many2one('account.invoice', 'Agent Invoice')
+    agent_invoice_id = fields.Many2one(
+        'account.invoice', 'Agent Invoice', readonly=True)
```
Comment (JayVora-SerpentCS) [inline account_fiscal_agent/models/account_invoice.py:13]: This is we added,(was not a part of the doc) to create a link between the 2 invoices.

Comment (eLBati) [reply account_fiscal_agent/models/account_invoice.py:13]: Right, well done

File: account_fiscal_agent/models/account_invoice.py [hunk 3]
```diff
@@ -55,26 +53,13 @@ def invoice_validate(self):
...
-                        invoice_line.partner_id.id,
-                        invoice_line.invoice_id.fiscal_position and
-                        invoice_line.invoice_id.fiscal_position.id or False,
-                        invoice_line.price_unit,
-                        invoice_line.invoice_id.currency_id and
-                        invoice_line.invoice_id.currency_id.id or False,
-                        invoice_line.company_id.id)
-                invoice_line_vals.update(product_onchange_vals.get('value'))
-                invoice_line_vals.update({
+                invoice_line_vals = {
```
Comment (JayVora-SerpentCS) [inline account_fiscal_agent/models/account_invoice.py:60]: Do not we need an onchange?

**Pull request merged**